Editor/Gui/Windows/ScreenManager.cs

```diff
@@ -0,0 +1,441 @@
+using System.Diagnostics;
+using System.Drawing;
+using System.Windows.Forms;
+using T3.Core.Utils;
+using T3.Editor.App;
+using T3.Editor.Gui.Input;
+using T3.Editor.Gui.Styling;
+using T3.Editor.Gui.UiHelpers;
+using T3.Editor.Gui.Windows.Layouts;
+using Icon = T3.Editor.Gui.Styling.Icon;
+using Vector2 = System.Numerics.Vector2;
+
+namespace T3.Editor.Gui.Windows;
+
+internal sealed class ScreenManager : Window
+{
+    internal ScreenManager()
+    {
+        Config.Title = "Screen Manager";
+        SystemEvents.DisplaySettingsChanged += (_, __) => _layoutDirty = true;
+    }
+
+    protected override void DrawContent()
+    {
+        FormInputs.AddVerticalSpace(15);
+
+        DrawInnerContent();
+    }
+
+    internal override IReadOnlyList<Window> GetInstances()
+    {
+        throw new NotImplementedException();
+    }
+
+    private static void DrawInnerContent()
+    {
+        RefreshScreenCache();
+        ImGui.Indent(10);
+        FormInputs.AddVerticalSpace(10);
+
+        var windowWidth = ImGui.GetWindowWidth();
+
+        // Mark layout dirty if window width changed
+        if (Math.Abs(windowWidth - _lastWindowWidth) > 0.5f)
+        {
+            _layoutDirty = true;
+            _lastWindowWidth = windowWidth;
+        }
+
+        if (_layoutDirty)
+        {
+            _cachedScale = ComputeScale(_cachedOverallBounds, windowWidth);
+            _layoutDirty = false;
+        }
+        ImGui.Unindent(10);
+        DrawScreenLayout(_cachedScreens, _cachedScale);
+
+        ImGui.Indent(10);
+        FormInputs.AddVerticalSpace(20);
+
+        if (ImGui.Button("Windows Display settings" + new string(' ', 5)))
+            OpenWindowsDisplaySettings();
+
+        Icons.DrawIconOnLastItem(Icon.OpenExternally, UiColors.Text, .99f);
+        CustomComponents.TooltipForLastItem("Open Windows display settings to configure screen arrangement, resolution, etc.");
+        ImGui.Dummy(new Vector2(1, 0));
+        ImGui.SameLine(windowWidth - 30);
+        ImGui.PushStyleColor(ImGuiCol.Text, UiColors.TextMuted.Rgba);
+        ImGui.PushFont(Icons.IconFont);
+        Icons.DrawAtCursor(Icon.Tip);
+        ImGui.PopFont();
+        ImGui.PopStyleColor();
+        CustomComponents.TooltipForLastItem("Press F11 twice to update the UI position");
+
+        FormInputs.AddVerticalSpace(20);
+        
+        
+
+        ImGui.Unindent(10);
+
+        if (_spanningNeedsUpdate)
+        {
+            ApplySpanningChanges();
+            _spanningNeedsUpdate = false;
+        }
+    }
+
+
+    private static void ApplySpanningChanges()
+    {
+        var spanningBounds = UserSettings.Config.OutputArea;
+
+        // Check if spanning area is defined
+        if (spanningBounds.Z > 0 && spanningBounds.W > 0)
+        {
+            // Enable the output window if not already enabled
+            if (!WindowManager.ShowSecondaryRenderWindow)
+            {
+                WindowManager.ShowSecondaryRenderWindow = true;
+            }
```

Comment:
Why check if you can just set?
```suggestion
WindowManager.ShowSecondaryRenderWindow = true;
```